nagios/datadog_checks/nagios/nagios.py

```diff
@@ -64,10 +73,23 @@ class Nagios(AgentCheck):
         re.compile('^(?P<key>service_perfdata_file)\s*=\s*(?P<value>.+)$'),
     ]
 
+    # Wrapper for gauge functions that do not send timestamps
+    def timestamp_excluded_gauge(self, *args, **kwargs):
```

Comment:
I think this could be simply `def gauge(self, *args, **kwargs):` calling `super(Nagios, self).gauge` in the body and you would save lines 88 to 91

Reply:
Correct, even if the lookup of a key in a dictionary is negligible in my opinion.

Reply:
Excellent :D